tests/ert/unit_tests/config/test_observations.py

```diff
@@ -1703,3 +1704,27 @@ def test_that_setting_an_unknown_key_in_a_segment_is_not_valid():
                 };
             };
         """)
+
+
+def test_ert_config_logs_observation_types_and_keywords(caplog):
+    obs_config_contents = """
+    GENERAL_OBSERVATION OBS1 {
```

Comment:
Maybe put in `HISTORY_OBSERVATION FOPR;` and a use of `SEGMENT` into this test, then you have to use `make_refcase_observations` and add the `use_tmpdir` fixture.